database/src/main/java/entities/Tag.java

```diff
@@ -11,11 +10,12 @@
 import java.io.Serializable;
 
 @Entity
+@ToString
```

Comment:
Do we really want the Lombok's ToString here? You know it's format and it won't be good in terms of viewing